src/main/java/com/mariana/gitworkflow/GitWorkflow.java

```diff
@@ -0,0 +1,14 @@
+package com.mariana.gitworkflow;
+
+public class GitWorkflow {
+    public static void main(String[] args ) {
+
+        String step1 = "Perform git stash - to store the uncommitted changes";
+
+
```

Comment:
There are missing a lot of details

Reply:
Hello! Please review the new commit. For clarity, its message is "Defined the Git WorkFlow for future development strategy".